packages/shared/src/contexts/AuthContext.tsx

```diff
@@ -17,7 +17,7 @@ import { AccessToken, Boot, Visit } from '../lib/boot';
 import { isCompanionActivated } from '../lib/element';
```

Comment:
Generic thread:
Should we not also fix the implementation changes that used `!isAuthReady`

Reply:
That has been reverted already in the fix for hydration.